public/util/types.ts

```diff
@@ -375,6 +375,10 @@ export function isImageType(type: string): boolean {
   return IMAGE_TYPES.indexOf(type) !== -1;
 }
 
+export function isRemoteSensingType(type: string): boolean {
```

Comment:
We're trying to get rid of "remote sensing" as a term in the client, since the actual var type is multiband image.  I would rename this to `isMultibandImageType`.